### ISSUE TITLE
charts that need to run on master nodes will need to add tolerations

### ISSUE THREAD

**Issue by DStorck** [samsung-cnct/k2-charts#107]
once https://github.com/samsung-cnct/k2/pull/588 is merged, any charts that need to run in the master nodepool will need to include tolerations for this master taint. 

example pod.yaml: 
```yaml
apiVersion: v1
kind: Pod
metadata:
  name: {{.Values.name}}
  labels:
    app: log-gen
spec:
  containers:
    - name: log-gen
      image: {{.Values.image}}
      ports:
      - containerPort: 8080
    {{- if .Values.tolerations }}
  tolerations:
{{ toYaml .Values.tolerations | indent 4 }}
      {{- end}}
```

example values.yaml: 
```yaml

name: log-gen
image: quay.io/dstorck/log_app:latest
tolerations:
 - key: node-role.kubernetes.io/master
   operator: Equal
   effect: NoSchedule
 - operator: Exists
```

**Comment by DStorck** [comment]
Current plan: implement option for tolerations on all charts, at least that way the user can pass in values to create a toleration if desired.  Add toleration for master taint to daemonsets.

**Comment by DStorck** [comment]
currently adding tolerations to: 
- heapster
- fluentd-daemonset
- gci-iptable-conf-agent
and adding empty array and loop to all over replicasets, daemonsets, pods, and statefulsets

**Comment by DStorck** [comment]
PR for master node tolerations: https://github.com/samsung-cnct/k2-charts/pull/111

new ticket for adding toleration option to charts: https://github.com/samsung-cnct/k2-charts/issues/112

**Comment by leahnp** [comment]
LGTM
